mlir/examples/standalone/python/StandaloneExtensionPybind11.cpp

```diff
@@ -12,9 +12,11 @@
 #include "Standalone-c/Dialects.h"
 #include "mlir/Bindings/Python/PybindAdaptors.h"
 
+namespace py = pybind11;
+
 using namespace mlir::python::adaptors;
 
-PYBIND11_MODULE(_standaloneDialectsPybind11, m) {
+PYBIND11_MODULE(_standaloneDialects, m, py::mod_gil_not_used()) {
```

Comment:
Again I think this should be `_standaloneDialectsPybind11`, because we're building both pybind11 and nanobind versions of the same module.